src/picknik_ur_base_config/config/moveit/ur_servo.yaml

```diff
@@ -18,24 +18,26 @@ scale:
 override_velocity_scaling_factor: 0.5
 
 ## Properties of outgoing commands
-publish_period: 0.002  # 1/Nominal publish rate [seconds]
+publish_period: 0.01  # 1/Nominal publish rate [seconds]
+max_expected_latency: 0.1 # delay between sending a command and the robot executing it [seconds]
 
 # What type of topic does your robot driver expect?
 # Currently supported are std_msgs/Float64MultiArray or trajectory_msgs/JointTrajectory
-command_out_type: std_msgs/Float64MultiArray
+command_out_type: trajectory_msgs/JointTrajectory
 
 # What to publish? Can save some bandwidth as most robots only require positions or velocities
 publish_joint_positions: true
-publish_joint_velocities: false
+publish_joint_velocities: true
 publish_joint_accelerations: false
 
 ## Plugins for smoothing outgoing commands
 use_smoothing: true
-smoothing_filter_plugin_name: "online_signal_smoothing::ButterworthFilterPlugin"
-low_pass_filter_coeff: 1.5  # Larger --> trust the filtered data more, trust the measurements less.
+smoothing_filter_plugin_name: "online_signal_smoothing::AccelerationLimitedPlugin"
```

Comment:
Can you add a small comment in the PR description that this PR depends on the moveit2 PR that adds this and requires an dependency update of moveit2 in studio